### ISSUE TITLE
Lint::Debugger does not find `Kernel.binding.pry`

### ISSUE THREAD

**Issue by mobilutz** [bbatsov/rubocop#4646]
## Expected behavior

When writing `Kernel.binding.pry` instead of `binding.pry` the Cop `Lint::Debugger` should raise an error.

## Actual behavior

There is no violation being raised.

## Steps to reproduce the problem

Write `Kernel.binding.pry` instead of `binding.pry` in any source file.

## RuboCop version

```
$ rubocop -V
0.49.1 (using Parser 2.4.0.0, running on ruby 2.3.4 x86_64-darwin16)
```
